packages/block-library/src/post-comments/edit.js

```diff
@@ -135,16 +139,25 @@ export default function PostCommentsEdit( {
 												width="32"
 												loading="lazy"
 											/>
-											<b className="fn">
-												<a href="#top" className="url">
-													{ __(
-														'A WordPress Commenter'
-													) }
-												</a>
-											</b>{ ' ' }
-											<span className="says">
-												{ __( 'says' ) }:
-											</span>
+											{ createInterpolateElement(
+												__(
+													'<b><a>A WordPress Commenter</a></b> <span>says:</span>'
```

Comment:
Is it possible to use something like this to closer match the [WP Core strings](https://github.com/WordPress/wordpress-develop/blob/3fdce9eb4b13ab9e9f52c5636b730aaa43740b12/src/wp-includes/class-walker-comment.php#L438-L442)? 

```js
sprintf(
  /* translators: %s: Comment author link. */
  __( '%s <span>says:</span>' ),
  sprintf( '<cite><a>%s</a></cite>', __( 'A WordPress Commenter' ) )
)
```

I think the answer might be no because of how `createInterpolateElement()` is intended to work but I'd rather ask.

Reply:
If we don't add any class to the translation string, `createInterpolateElement()` works fine. 🎉